src/components/CommentContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+
+const CommentContainer = ({ article_id }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetch(
+      `https://project1-be-nc-news.onrender.com/api/articles/${article_id}/comments`
+    )
+      .then((response) => response.json())
+      .then((response) => {
+        console.log(response);
+        setComments(response.comments);
+      })
+      .catch(() => {
+        setIsError(true);
+        setIsLoading(false);
+      });
+  }, [article_id]);
+
+  if (isError) return <p>This is awkward, I cant load the comments...</p>;
```

Comment:
I think you need to check article 37. It has no comments but receives this error text. Is an article with no comments an error? What about new articles, no one could possibly comment on it so wouldn't have any.